content/en/actions/app_builder/build.md

```diff
@@ -16,70 +16,131 @@ further_reading:
 
 You can create an app or edit existing apps from the [App Builder][1] page. The page lists information about existing apps, including the following:
 - Author
-- Status
+- Tags
 - Date that each app was last modified
-- Whether the app is published
+- Whether each app is published
 
 On the App Builder page, you can access and filter your apps. Hover over an app for options to edit, delete, view, or clone the app. You can also enable the **My apps** toggle to see only apps that you created:
 
 {{< img src="service_management/app_builder/app-builder-my-apps-2025-11-19.png" alt="The App Builder page, filtered to show only 'My apps'" style="width:100%;" >}}
 
-## Create an app
-
-### Build an app from a blueprint
+## Start with onboarding
 
-Blueprints are helpful starter apps that cover common use cases. They come loaded with demo data that you can use to familiarize yourself with the app. Blueprints also showcase best practices for setting up app functionality and visual presentation.
+When creating a new app, Datadog displays an onboarding modal with the following options: 
+- [Start with AI][22]
+- [Start with blueprint][18]
+- [Start with layout][19]
+- [Start with data][20]
+- [Start from scratch][21] 
+- [App Playground][23]
```

Comment:
I noticed that you cover this step in each of thh separate methods too, and it can be disorienting to users if they see the same step detailed twice (they might think they have to do the same thing more than once, or wonder if they clicked into the wrong place, or something like that). You could frame this instead as different methods of creating a new app, so there's less confusion about where the procedure actually starts.